### ISSUE TITLE
[FEAT] show human readable values

### ISSUE THREAD

**Issue by uhthomas** [AnalogJ/scrutiny#570]
**Is your feature request related to a problem? Please describe.**

Values like 2984964136 and 923248607 are pretty meaningless and hard to parse as a human.

![image](https://github.com/AnalogJ/scrutiny/assets/9749173/434eafb1-f0c1-4da7-99c0-a202106d6258)

**Describe the solution you'd like**

It would be nice if the UI could this this in a more human-like form. Such 2.98PB and 9.23PB.

**Additional context**

Some disks have huge numbers of data written and read. It's very difficult (if not impossible) to understand whether a number is gigabytes, terabytes or petabytes at a glance.

**Comment by uhthomas** [comment]
I'd like to extend this request to also include values like 'power on hours'.

![image](https://github.com/AnalogJ/scrutiny/assets/9749173/62dd3ede-ad29-4fcd-9dda-a0c07310b0b3)

Ideally, this would be represented as `35,050`, or `1460 days and 10 hours`, or `4 years`.

It may be best to have both values? Such `35,050 hours (4 years)`.

In the case of units read/written: `2,984,964,136 bytes (2.98PB)`.

**Comment by kaysond** [comment]
This issue is being closed automatically as a part of an effort to streamline and prioritize development effort. If it is still relevant, please reply to this comment and we will convert it to a discussion.
